reasoner/controller/ConcludableController.java

```diff
@@ -248,9 +258,27 @@ protected void mayAddTraversal() {
             private FunctionalIterator<ConceptMap> filterInferred(ConceptMap conceptMap) {
                 // TODO: Requires duplicate effort to filter out inferred concludable answers. Instead create a new
                 //  traversal mode that doesn't return inferred concepts
-                if (concludable.isInferredAnswer(conceptMap)) return Iterators.empty();
+                if (concludable.isInferredAnswer(conceptMap)) return empty();
                 for (Concept c : conceptMap.concepts().values()) {
-                    if (c.isThing() && c.asThing().isInferred()) return Iterators.empty();
+                    if (c.isThing() && c.asThing().isInferred()) return empty();
+                }
+                return Iterators.single(conceptMap);
+            }
+
+            @Override
+            protected FunctionalIterator<ConceptMap> rejectMismatchedInference(ConceptMap conceptMap) {
+                Concept conceptToCheck = null;
+                if (concludable.isAttribute()) {
+                    conceptToCheck = conceptMap.get(concludable.asAttribute().attribute().id());
+                } else if (concludable.isIsa()) {
+                    conceptToCheck = conceptMap.get(concludable.asIsa().isa().owner().id());
+                }
+                if (conceptToCheck != null) {
+                    if (conceptToCheck.asThing().isInferred()) {
+                        return Iterators.single(conceptMap);
+                    } else {
+                        return empty();
+                    }
```

Comment:
our style for these short `if` blocks is not to have `{}` and inline the `if` with the body:
```
if (condition) return ...;
else return empty()
```
Generally the rule is that if the `if (condition)` together with the body fits in one line, we don't need the `{}` or newlines